osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -90,12 +96,31 @@ private double strainValueOf(DifficultyHitObject current)
 
                     wideAngleBonus *= angleBonus * (1 - Math.Min(wideAngleBonus, Math.Pow(calcWideAngleBonus(lastAngle), 3))); // Penalize wide angles if they're repeated, reducing the penalty as the lastAngle gets more acute.
                     acuteAngleBonus *= 0.5 + 0.5 * (1 - Math.Min(acuteAngleBonus, Math.Pow(calcAcuteAngleBonus(lastLastAngle), 3))); // Penalize acute angles if they're repeated, reducing the penalty as the lastLastAngle gets more obtuse.
-
-                    angleBonus = acuteAngleBonus * acute_angle_multiplier + wideAngleBonus * wide_angle_multiplier; // add the angle buffs together.
                 }
             }
 
-            aimStrain += angleBonus; // Add in angle bonus.
+            if (Math.Max(prevVelocity, currVelocity) != 0)
+            {
+                prevVelocity = (osuLastObj.JumpDistance + osuLastObj.TravelDistance) / osuLastObj.StrainTime; // We want to use the average velocity over the whole object when awarding differences, not the individual jump and slider path velocities.
```

Comment:
These lines are too long. put the comments above and potentially split over multiple lines.

Reply:
Resolved in velocity PR